src/main/java/io/fabric8/jenkins/openshiftsync/JenkinsUtils.java

```diff
@@ -545,9 +555,11 @@ public static void deleteRun(WorkflowRun run) {
 			}
 	}
 
+
   public static void deleteRun(WorkflowJob job, Build build) {
       WorkflowRun run = getRun(job, build);
```

Comment:
probably, we should set the deleteRunIfPruneEnabled method call the deleteRun method, and not the opposite.
Let's discuss it.

Reply:
We would need the BuildConfigProjectProperty in this case which I wasn't able to get using the Run. Hence the wrapper.